src/styles/_mixins.scss

```diff
@@ -0,0 +1,4 @@
+@mixin card-size {
+  height: $card-height;
+  width: $card-width;
```

Comment:
Actually there is no need to set width to card, it's enough to set it only to column, card should take all column width